spark-postgres/src/test/scala/io/frama/parisni/spark/postgres/DdlTest.scala

```diff
@@ -951,10 +951,10 @@ class DdlTest extends QueryTest with SparkSessionTestWrapper {
       "FLOAT_COL",
       "DOUBLE_COL",
       "STRING_COL",
-      "DATE_COL",
-      "TIMESTAMP_COL",
+      //"DATE_COL",
```

Comment:
do you have some context for commenting this ?

Reply:
Problem is that with Scala 2.12+ it becomes impossible to have in `Dataset` columns with types `Array(Some(X))`, where `X` is `java.sql.Date`, `java.sql.Timestamp`, `scal.math.BigDecimal`. E.g the following code fragment 
```scala
Seq(Array(Some(testDate))).toDS()
```
causes error `org.apache.spark.SparkRuntimeException: Error while encoding: java.lang.RuntimeException: scala.Some is not a valid external type for schema of date` .

I commented the problematic lines, because, to be honest, I'm not sure how to fix this properly. Any suggestions?